src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -240,6 +240,8 @@
             <argument>%api_platform.title%</argument>
             <argument>%api_platform.description%</argument>
             <argument>%api_platform.version%</argument>
+            <argument>null</argument>
+            <argument on-invalid="null">%api_platform.swagger.versions%</argument>
```

Comment:
```suggestion
            <argument>%api_platform.swagger.versions%</argument>
```

This parameter is always defined isn't it?

Reply:
no, it's not.
As soon as the configuration swagger version is an empty array, this parameter is not set.
This follow the same logic as for the `api_platform.swagger.api_keys` parameter.